src/DisallowedActionsSerializer.js

```diff
@@ -6,7 +6,7 @@
 */
 import type { ActionToggleRegister } from './types';
 
-export default class AllowedActionsSerializer {
+export default class DisallowedActionsSerializer {
```

Comment:
The source code for `DisallowedActionsSerializer` contains a few references to 'allowed' rather than 'disallowed' that need to be updated.

Such as the file level comment:

```
We use the same conventions as the program to store a register of "allowed actions" in the URL, so that the choices
made in the "simplification" menu are persisted.
```

I guess this comment should also be updated as "same conventions as the program" isn't strictly true any longer.

And some variables, such as:

```
let serializedAllowedActions = '';
for (const [actionKey, isAllowed] of Object.entries(actionToggleRegister)) {
    if (isAllowed) {
```

Reply:
I fixed these.  I also searched for, found, and fixed a few others.